test/githubWrapperTests.js

```diff
@@ -60,11 +79,114 @@ describe('githubwrapper.js Tests', function() {
 
     it('should return a promise from getUser', function() {
         var username = 'testUser';
-        var result = gitHubWrapper.getUser(username);
+        var result = gitHubWrapper.getUser(username, {});
 
         expect(result).to.have.property('then');
     });
 
+    it('authenticate should use token type auth when username is `<token>`', (done) => {
+        const password = 'thisisanapitoken';
+        sandbox.stub(inquirer, 'prompt').resolves({username: '<token>', password});
+        gitHubWrapper.authenticate().then(() => {
+            expect(mockGithub.authenticate.calledOnce).to.be.true;
+            expect(mockGithub.authenticate.args[0][0]).to.eql({type: 'token', token: password});
```

Comment:
:nit: Use `calledWith` rather than array syntax

Reply:
the reason I did this was because the error message for `expect(mockGithub.authenticate.calledWith({...}).to.be.true` is rather vague and doesn't explain much about what the difference is. This way we get a clear expected vs actual data comparison

Reply:
Ah, yes.  Now I remember that's why I use `sinon-chai`. 😄